web/satellite/src/utils/accessGrant.worker.js

```diff
@@ -87,4 +87,4 @@ self.onmessage = async function (event) {
     default:
         self.postMessage(new Error('provided message event type is not supported'));
     }
-};
+};
```

Comment:
I'm not sure why this changes appear. Seems like one empty line at the end of the file got removed